урок 10 ооп.ч1.py

```diff
@@ -0,0 +1,19 @@
+class Matrix:
+
+    def __init__(self, matrix):
+        self.matrix = matrix
+
+    def __str__(self):
+        return '\n'.join(map(lambda r: ' '.join(map(str, r)), self.matrix)) + '\n'
+
+    def __add__(self, other):
+        return Matrix(map(lambda  r_1, r_2: map(lambda x, y: x+y, r_1, r_2), self.matrix, other.matrix))
+
```

Comment:
a11 = int(input("input for a11 ") )
a12  = int(input("input for a12 "))
a13 = int(input("input for a13 "))
a21 = int(input("input for a21 "))
a22 = int(input("input for a22 "))
a23= int(input("input for a23"))
// and same for the second matrix which will make it user friendly